force-app/main/default/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -676,8 +677,8 @@ export default class rd2EntryForm extends LightningElement {
     async processCommitmentSubmit(allFields) {
         try {
             if (this.isElevateWidgetDisplayed()) {
-                this.loadingText = this.rd2Service.getPaymentProcessingMessage(this.rd2State.paymentMethod);
-
+                
+                this.loadingText = this.isExperienceSite && this.rd2Service.getPaymentProcessingMessage(this.rd2State.paymentMethod).includes(ELEVATE) ? '' : this.rd2Service.getPaymentProcessingMessage(this.rd2State.paymentMethod);
```

Comment:
This feels fragile to me. The code is looking for the word "elevate" in text to determine if Elevate is enabled or not (I believe). Is there a better way to determine if Elevate is enabled or not? Assuming that donors cannot create a new Recurring Donation in this UI, then I believe the RD.CommitmentId__c field will have a value if it's an Elevate RD. Would that be a more reliable check to determine if Elevate is enabled?

Reply:
We should not show the word 'elevate' in the spinners if the component is used from the experience sites. i was checking the custom label value, but I agree it's a better idea to check the RD.CommitmentId__c. Let me apply the update.

Reply:
@gerardorodriguezsf  Thanks. Is the intent to display nothing at all if "Elevate" is anywhere in the PaymentProcessing message?

Reply:
It' showing 'elevate' word only for ACH payments. I updated the condition to check the payment method.